### PR TITLE
[ZPI-26] Login page

### PR DESCRIPTION
**Pull request by gcfbn**

### CODE REVIEW

File: frontend/src/app/i18n/client.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }, [activeLng, i18n.resolvedLanguage])
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useEffect(() => {
+      if (!lng || i18n.resolvedLanguage === lng) return
+      i18n.changeLanguage(lng)
+    }, [lng, i18n])
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useEffect(() => {
+      if (cookies.i18next === lng) return
+      setCookie(cookieName, lng, { path: '/' })
```
Comment (Veermove) [inline frontend/src/app/i18n/client.ts:53]: so wait we are using cookies to track langauge, so does it need to be path segment?

File: frontend/src/app/i18n/locales/en/signin.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "signIn": "Sign In",
+    "email": "Email",
+    "password": "Password",
+    "incorrectEmail": "Incorrect email"
```
Comment (Veermove) [inline frontend/src/app/i18n/locales/en/signin.json:5]: i haven't looked further yet, but is it not missing something like `Login failed, incorrect password or email` ? That case can happen right?

File: frontend/src/components/isAuth.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+'use client';
+import { useEffect } from "react";
```
Comment (Veermove) [inline frontend/src/components/isAuth.tsx:2]: This whole file is unused

**Review by Veermove — CHANGES_REQUESTED** [review]

**Review by Remmo1 — COMMENTED** [review]
ok

**Review by Remmo1 — APPROVED** [review]

**Review by Veermove — APPROVED** [review]
Looks good, much cleaner components than before

**Pull request merged**